components/ten_years_ago_component.py

```
@@ -0,0 +1,34 @@
+import streamlit as st
+
+# 10년 전 오늘 데이터를 출력하는 컴포넌트입니다 ( data 호출은 따로 )
+# Manual
+# 1. df = get_10years_ago_news 호출 : 10년 전 기사 데이터 가져옴
+# 2. con(df) or sidebarCon(df) : 가져온 데이터로 컴포넌트 생성
+
+def get_10years_ago_news():
+
+    correct_time = datetime.datetime.now() - relativedelta(years=10)
```

Ruff (0.11.9)

Local variable `correct_time` is assigned to but never used

Remove assignment to unused variable `correct_time`

(F841)